Description: fix test
Author: Yadd <yadd@debian.org>
Bug-Debian: https://bugs.debian.org/1009601
Forwarded: no
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2022-04-13

--- a/test/test.js
+++ b/test/test.js
@@ -41,7 +41,7 @@
     it('should set Content-Type', function (done) {
       request(server)
         .get('/todo.txt')
-        .expect('Content-Type', 'text/plain; charset=UTF-8')
+        .expect('Content-Type', 'text/plain')
         .expect(200, done)
     })
 
